packages/webapp/src/containers/Map/index.jsx

```diff
@@ -488,13 +493,15 @@ export default function Map({ history }) {
         )}
         {showBulkSensorUploadModal && (
           <BulkSensorUploadModal
-            dismissModal={() => {
-              setShowBulkSensorUploadModal(false);
-              handleClickAdd();
-            }}
+            dismissModal={dismissBulkSensorsUploadModal}
             onUpload={(file) => {
-              console.log('file', file);
-              // call saga function here
+              const payload = {
+                file,
+                dismissBulkSensorsUploadModal: () => {
```

Comment:
Instead of this, can we use the loading state to figure out when to hide the modal?

Reply:
To keep are redux logic separate from our components.

Reply:
sure